<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SWCloud</class>
 <widget class="QMainWindow" name="SWCloud">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>410</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>410</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>SWCloud</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>SWCloud.ico</normaloff>SWCloud.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>380</width>
     <height>410</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>380</width>
     <height>410</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btnEnviar">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>240</y>
         <width>221</width>
         <height>41</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Montserrat Medium</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius: 13px;
background-color:  rgb(38,182,131);
border-style: outset;
color: rgb(255,255,255);
}
QPushButton:hover{
background-color:  rgb(21,180,140);
}
QPushButton:pressed {	
   background-color: rgb(255, 255, 255);
   color: rgb(38, 182, 131);
   border-color:  rgb(38, 182, 131);
   border-radius: 12px;
   border-style: outset;
   border-width: 2px;
}</string>
       </property>
       <property name="text">
        <string>Enviar</string>
       </property>
      </widget>
      <widget class="QLabel" name="versao">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>300</y>
         <width>381</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat Medium</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(96,96,96)</string>
       </property>
       <property name="text">
        <string>Versão 1.00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="build">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>320</y>
         <width>381</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat Medium</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(96,96,96)</string>
       </property>
       <property name="text">
        <string>Build 1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="projeto_Python">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>50</y>
         <width>381</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat SemiBold</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">	color: rgb(38, 182, 131);</string>
       </property>
       <property name="text">
        <string>PROJETO PYTHON</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="versao_Demo">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>80</y>
         <width>381</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat Light</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(101,101,101)</string>
       </property>
       <property name="text">
        <string>Versão Demo</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="texto">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>140</y>
         <width>381</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat Medium</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(101,101,101)</string>
       </property>
       <property name="text">
        <string>Clique em enviar para atualizar</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="texto2">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>170</y>
         <width>381</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat Medium</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(101,101,101)</string>
       </property>
       <property name="text">
        <string>os preços do seu site.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="ultimaCarga">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>370</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat Medium</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(96, 96, 96);
background-color: transparent;</string>
       </property>
       <property name="text">
        <string>Última carga:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="dateCarga">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>370</y>
         <width>271</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat Medium</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(96, 96, 96);
background-color: transparent;</string>
       </property>
       <property name="text">
        <string>16/09/2021 08:05</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <zorder>btnEnviar</zorder>
      <zorder>versao</zorder>
      <zorder>build</zorder>
      <zorder>versao_Demo</zorder>
      <zorder>projeto_Python</zorder>
      <zorder>texto</zorder>
      <zorder>texto2</zorder>
      <zorder>ultimaCarga</zorder>
      <zorder>dateCarga</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btnEnviar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
